modules/samples/producers/wso2-event-performance/src/main/java/org/wso2/carbon/sample/performance/Client.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.carbon.sample.performance;
+
+import org.apache.log4j.Logger;
+import org.wso2.carbon.databridge.agent.AgentHolder;
+import org.wso2.carbon.databridge.agent.DataPublisher;
+import org.wso2.carbon.databridge.commons.Event;
+
+import java.text.DecimalFormat;
+import java.util.Arrays;
+import java.util.Random;
+
+public class Client {
+    private static Logger log = Logger.getLogger(Client.class);
+
```

Comment:
Hi Tharik,

Since this is a performance test client, shall we consider these improvements as well? Suggesting these improvements since this was available in the previous version of the client AFAIR.

1. For performance tests, I think it is better if events are sent initially for a warm up period, then let the server idle for some time, and then start the actual test. If some parameters are available for these actions as well, it might be easier to run the performance tests. But for now, we can do it in a manual fashion if needed.
2. Shall we include System.nanoTime as well as an attribute of the stream. System.nanoTime cannot be used to compare time between receiver and publisher since it differs on different JVMs. But the gap between timestamps of two events might be used for some calculations, and if any calculations are performed at the receiver end that need accuracy, including nanoTime also might be useful. Just a suggestion in case this client is used for complex performance calculations...